### PR TITLE
phpbrew 1.27.0 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

File: Formula/phpbrew.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  homepage "https://phpbrew.github.io/phpbrew"
+  url "https://github.com/phpbrew/phpbrew/releases/download/1.27.0/phpbrew.phar"
+  sha256 "0fdcda638851ef7e306f5046ff1f9de291443656a35f5150d84368c88aa7a41a"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    chmod "+x", "phpbrew.phar"
+    bin.install "phpbrew.phar" => "phpbrew"
```
Comment (carlocab) [inline Formula/phpbrew.rb:12]: Doesn't `bin.install` set permissions correctly on its own?

Comment (SMillerDev) [reply Formula/phpbrew.rb:12]: Not if it wasn't executable to begin with I think.

**Review by SMillerDev — COMMENTED** [review]
I think this should be `uses_from_macos "php"`

**Comment by chenrui333** [comment]
> uses_from_macos "php"

Good call out.

File: Formula/phpbrew.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Phpbrew < Formula
+  desc "Brew & manage PHP versions in pure PHP at HOME"
+  homepage "https://phpbrew.github.io/phpbrew"
+  url "https://github.com/phpbrew/phpbrew/releases/download/1.27.0/phpbrew.phar"
+  sha256 "0fdcda638851ef7e306f5046ff1f9de291443656a35f5150d84368c88aa7a41a"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
```
Comment (chenrui333) [inline Formula/phpbrew.rb:10]:
```suggestion
  bottle :unneeded

  depends_on "php"

  def install
```

Comment (carlocab) [reply Formula/phpbrew.rb]: not `uses_from_macos`?

Comment (chenrui333) [reply Formula/phpbrew.rb]: I dont think we have `uses_from_macos` for php?

Comment (SMillerDev) [reply Formula/phpbrew.rb]: We should, macOS provides php

**Review by SMillerDev — APPROVED** [review]

**Comment by carlocab** [comment]
@dawidd6 you forgot to use `--autosquash` 😅

**Pull request closed without merging**